File: plugin/src/org/jetbrains/haskell/compiler/GHCMessage.java
package org.jetbrains.haskell.compiler;

/**
 * @author Evgeny.Kurbatsky
 */
public class GHCMessage {
    public final String myFile;
    public  final int myLine;
    public final int myCol;
    public String myText;

    public GHCMessage(String file, String line, String col) {
        myFile = file;
        myLine = Integer.parseInt(line);
        myCol = Integer.parseInt(col);
    }

    public void setText(String text) {
        myText = text;
    }
}

File: plugin/src/org/jetbrains/haskell/compiler/GHCMessage.kt
package org.jetbrains.haskell.compiler

/**
 * @author Evgeny.Kurbatsky
 */
public class GHCMessage(file: String, line: String, col: String, text: String) {
    public val myFile: String = file
    public val myLine: Int = Integer.parseInt(line)
    public val myCol: Int = Integer.parseInt(col)
    public var myText: String = text;
}
